src/sonic-build-hooks/scripts/collect_version_files

```diff
@@ -7,7 +7,7 @@ TARGET_PATH=$1
 [ -z "$TARGET_PATH" ] && TARGET_PATH=$POST_VERSION_PATH
 ARCH=$(dpkg --print-architecture)
 DIST=$(grep VERSION_CODENAME /etc/os-release | cut -d= -f2)
-([ -z "$DIST" ] && grep -q jessie /etc/os-release) && DIST=jessie
```

Comment:
I think this line can actually be removed; it looks like Jessie didn't have the `VERSION_CODENAME` field, so this was added because of that.

Reply:
Thanks, fixed!